src/pruner_util.h

```diff
@@ -0,0 +1,27 @@
+#ifndef _PRUNER_UTIL_H_
+#define _PRUNER_UTIL_H_
+
+#include "ir/ir.h"
+
+#include "pruner_options.h"
+
+#define INFO(x) std::cout << x << std::endl;
+
+// define some fixed constants
+#define PRUNE_STMT_MAX 100
+#define PRUNE_ITERS 50
+#define NO_CHNG_ITERS 7
+cstring STRIPPED_NAME;
```

Comment:
STRIPPED NAME is now a global variable which we set at the beginning since it remains the same throughout the entire execution. Saves a little bit of redundant code pain.